packages/mermaid-mindmap/src/detector.ts

```diff
@@ -1,10 +1,20 @@
-export const id = 'mindmap';
+import type { ExternalDiagramDefinition } from 'mermaid';
 
-export const detector = (txt: string) => {
+const id = 'mindmap';
+
+const detector = (txt: string) => {
   return txt.match(/^\s*mindmap/) !== null;
 };
 
-export const loadDiagram = async () => {
+const loader = async () => {
```

Comment:
I removed it to keep the package in line with the "official" way of loading diagrams. 

_This needs to be handled in documentation, but we first need the API to be stable._

We should make it clear that the only thing needed in an external diagram is the `ExternalDiagramDefinition`. The extra imports aren't useful to anyone who just wants to add an external diagram. 
In case anyone really wants to access them, they can simply use `const {id, detector, loader} = mindmap`.

Also, when there is more than one diagram, the import statement gets ugly. I'm usually not a fan of default exports, but this is one scenario where it makes sense. 

```js
import {
  loader as mindmapLoader, detector as mindmapDetector, id as mindmapId
} from "@mermaid-js/mermaid-mindmap";
```

```js
import mindmap from "@mermaid-js/mermaid-mindmap";
// mindmap.id
// mindmap.loader
// mindmap.detector
```

Reply:
> I removed it to keep the package in line with the "official" way of loading diagrams.

Sounds good. It's pretty unlikely anybody was using the old syntax, but just in case, we should probably document this in the CHANGELOG/Release notes, since it is technically a breaking change.